### DISCUSSION TITLE
关于快捷键扩展的建议

### DISCUSSION THREAD

**Discussion by FoundTheWOUT** [the1812/Bilibili-Evolved#3280] [功能建议 / Ideas]
可否给每个快捷键加个开关？
这个想法由 #3247 引发。
挺喜欢这个插件，但有些快捷键可能更喜欢原生（比如arrowright这些，原生里长按是加速，扩展里是阶级快进，都有存在的意义，但不同人可能有不同的需求），还可以规避一些比较难处理的问题。

**Comment by the1812** [comment]
自定义按键 + 留空
